topics/scicloud.qmd

```diff
@@ -0,0 +1,36 @@
+---
+title: SciCloud
+---
+IT for Research (ITvO) offers SciCloud, a service where server capacity can be purchased to support research applications. SciCloud virtual servers are easy to implement, easy to scale up and down and have competitive pricing based on agreed purchase. The virtual server can be placed in an external network so services running on it can be made internet accessible. It can also run on an internal network for easy security and access to some VU services.
+
+SciCloud is based on the open source middleware [OpenNebula (ONE)](https://opennebula.io/) and offers a service that allows users to quickly start with a virtual server based on available templates, or to upload one of their own virtual images. This concept is known as Infrastructure as a Service (IaaS).
+
+Purchase of virtual server capacity takes place according to standard building blocks, combinations of the following two options can be purchased: 
+
+• Building Block 1: 1 CPU core with 2 GB RAM 
+
+• Building Block 2: 1 CPU core with 4 GB RAM 
+<!-- add a link to page on costs somewhere (? haven't found it yet) --> 
+
+Different configurations are possible in consultation and based on availability.  
+ 
+You are responsible for installing and maintaining any applications on the server yourself. On a consultancy basis, ITvO can assist in the configuration and optimization of the virtual server.
+
+## SciCloud functionalities 
+- A virtual server with guaranteed CPU power and reserved RAM.
+- Full administrator (root) access, so that the server can be adapted to your own wishes.
+- A choice of templates with Windows and Linux operating systems. It is also possible to install your own OS, license and security permitting.
+- Linking to shares on SciStor is possible.
+- Monitoring of operation and used capacity of the virtual server through a web portal.
+- Transparent cost, you pay a fixed monthly fee based on the configuration via internal billing.
+
+## Storage 
+By default, the virtual server is provided with 30 GB of storage capacity. Do you need more storage? Then a link with [SciStor](./scistor.qmd) storage is a possibile providing access to fast networked storage. ITvO can advise on this.
+
+## Backup 
+It is possible to take _snapshots_ or _clones_ of the virtual server yourself. This allows you to go back to a previously created configuration, if necessary. In addition, ITvO makes daily backups with which a virtual machine can be restored up to a month ago on request. 
+
+## Requesting process
+Via form on [ServiceNow](https://services.vu.nl/esc), go to: IT > My work field > Research > SciCloud > Order SciCloud capacity
```

Comment:
Would it be useful to add another section 'Contact'? The ServiceNow form is mentioned for requesting access, but perhaps it's good to explicitly explain how researchers can get in touch with ITvO if they have questions about SciCloud (if they just want more information, or if they encounter problems when using it), referring both to the ITvO email address and the possibility to ask a question through ServiceNow (I guess just with the general IT contact form)? What do you think, @peer35?

Reply:
As discussed I put in a mailto link for now.